docs/clusters/santis.md

```diff
@@ -1,3 +1,127 @@
 [](){#ref-cluster-santis}
 # Santis
 
+Santis is an Alps cluster that provides GPU accelerators and file systems designed to meet the needs of climate and weather models for the [CWp][ref-platform-cwp].
+
+## Cluster specification
+
+### Compute nodes
+
+Santis consists of around ??? [Grace-Hopper nodes][ref-alps-gh200-node].
```

Comment:
```suggestion
Santis consists of around 600 [Grace-Hopper nodes][ref-alps-gh200-node].
```
? Not sure what the official number is supposed to be. The table below says 1200, but I think the real number is lower than that at least.

Reply:
I have added a temporary link to the Gordon Bell guide.